### PR TITLE
[docs] Add TP and EA

### PR DESCRIPTION
**Pull request by ddhodge**
Add descriptions of tech preview and early access

@netlify /preview/releases/versioning/

### CODE REVIEW

File: docs/content/preview/releases/versioning.md [hunk 1]
```diff
@@ -51,3 +51,75 @@ Patch releases in the preview release series (`MAJOR.ODD.PATCH`) introduce new f
...
+
+## Feature availability
+
+YugabyteDB features are made available as one of the following:
+
+- Tech Preview (TP)
+- Early Access (EA)
+- General Availability (GA)
+
+Changes for EA and GA are always reported in the [YugabyteDB Release Notes](../release-notes/).
```
Comment (aishwarya24) [inline docs/content/preview/releases/versioning.md:63]:
```suggestion
Changes for EA and GA are always reported in the [YugabyteDB Release Notes](../releases/).
```

Comment (aishwarya24) [reply docs/content/preview/releases/versioning.md:63]: thats a broken link and if at all we report changes , I think it would be the releases homepage.

**Review by aishwarya24 — APPROVED** [review]
suggestion for a broken link.

File: docs/content/preview/releases/versioning.md [hunk 2]
```diff
@@ -52,3 +52,78 @@ Patch releases in the preview release series (`MAJOR.ODD.PATCH`) introduce new f
...
+
+## Feature availability
+
+YugabyteDB features are made available as one of the following:
+
+- Tech Preview (TP)
+- Early Access (EA)
+- General Availability (GA)
+
+Changes for EA and GA are always reported in the [YugabyteDB Release Notes](../releases/).
```
Comment (aishwarya24) [inline docs/content/preview/releases/versioning.md:64]: broken link

File: docs/content/preview/integrations/cdc/debezium.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Debezium and CDC in YugabyteDB
 linkTitle: Debezium
 description: Using Debezium for CDC in YugabyteDB.
-earlyAccess: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+earlyAccess: /preview/releases/versioning/#feature-availability
```
Comment (aishwarya24) [inline docs/content/preview/integrations/cdc/debezium.md:5]: why not link it to Earky access instaed? - /preview/releases/versioning/#early-access-ea

**Review by aishwarya24 — COMMENTED** [review]
The YSQL Connection Manager page also has a TP tag. : https://docs.yugabyte.com/preview/explore/connection-manager/connection-mgr-ysql/

File: docs/content/preview/integrations/cdc/debezium.md (hunk 3 above)
Comment (ddhodge) [reply docs/content/preview/integrations/cdc/debezium.md:5]: Thought about it, but felt it was useful to give all the context.

Comment (aishwarya24) [reply docs/content/preview/integrations/cdc/debezium.md:5]: yes fair point and actually easy maintaining it across too!

**Pull request merged**